api/projects/migrations/0009_project_enable_dynamo_db.py

```diff
@@ -6,13 +6,16 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('projects', '0008_project_hide_disabled_flags'),
+        ("projects", "0008_project_hide_disabled_flags"),
     ]
 
     operations = [
         migrations.AddField(
```

Comment:
I think this will need to be a separate AlterField migration?

Reply:
I don't think help_text have any effect on database(postgres at least), so creating another migration for that seems futile?

Reply:
I don't see why we wouldn't though? Django makes it so easy to create migrations and it's a bad habit to get into - updating old, applied migrations.